sys/include/net/loramac.h

```diff
@@ -102,6 +102,11 @@ extern "C" {
  * @name    LoRaMAC default values
  * @{
  */
+/**
+ * @brief Default LoRaWAN region
+ */
+#define LORAWAN_REGION_EU_868                  (true)
```

Comment:
Note that the prefix of these functions here is `LORAMAC`. I would suggest to change this symbol to `LORAMAC_REGION_EU_868`

Reply:
note that this hardcodes the region to EU 868. There should be guarded by an `#ifdef`

Reply:
Definitely yes. Need your suggestion on the best possible method to so that we can implement additional channels as well. Should I have a symbol LORAMAC_REGION_ENABLE ?

Reply:
```suggestion
#if !IS_ACTIVE(LORAWAN_REGION_EU_868) && !IS_ACTIVE(LORAWAN_REGION_IN_865)
#define LORAWAN_REGION_EU_868                  (true)
#endif
```

Reply:
Okay. But the list could get a whole lot bigger once we add additional channels. I have added a macro `LORAMAC_REGION_ACTIVE` to take care of the same. Hope that works.